addOns/fuzz/src/main/zapHomeFiles/jbrofuzz/fuzzers.jbrf

```diff
@@ -7,7 +7,7 @@
 #
 
 P:001-HTT-PMT:HTTP Methods:15
-> HTTP | Replacive Fuzzers 
+> HTTP | Replacing Fuzzers 

```

Comment:
How do we want to do this? Move it to commonlib and make the other add-ons depend on that?

Reply:
I'd suggest reverting them for the time being and handle in follow up PR.

I'd prefer not introduce an unnecessary dependency, but I guess sooner or later they will have to depend on `commonlib` (`fuzz` will very soon because of the thread changes so…).

Reply:
Okay